AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -312,13 +320,20 @@
 - (void)didExitHierarchy ASDISPLAYNODE_REQUIRES_SUPER;
 
 /**
- * Provides an opportunity to clear backing store and other memory-intensive intermediates, such as text layout managers
- * or downloaded content that can be written to a disk cache.
+ * Provides an opportunity to clear backing store and other memory-intensive intermediates, such as text layout managers.
  *
- * @discussion Called by -recursivelyReclaimMemory.   Base class implements self.contents = nil, clearing any backing
+ * @discussion Called by -recursivelyClearRendering. Base class implements self.contents = nil, clearing any backing
  * store, for asynchronous regeneration when needed.
  */
-- (void)reclaimMemory ASDISPLAYNODE_REQUIRES_SUPER;
+- (void)clearRendering ASDISPLAYNODE_REQUIRES_SUPER;
+
+/**
+ * Provides an opportunity to clear any remote data for only the current node.
```

Comment:
minor nit:  saying "only the current node" here but not on `-clearRendering` makes it look like that's unique to `-clearRemoteData`, maybe tweak the wording a bit?